app/src/main/java/org/mozilla/tv/firefox/ScreenController.kt

```diff
@@ -150,6 +151,17 @@ class ScreenController {
         transaction.commit()
     }
 
+    fun dispatchKeyEvent(keyEvent: KeyEvent, fragmentManager: FragmentManager): Boolean {
```

Comment:
I'm not super excited that we're adding another stop for key events (they already bounce around a lot), but this does seem like a logical way to organize it.  No action needed on this, just general grumbling.

Reply:
Better here than in `WebRenderFragment`, too.